scripts/local_testnet/setup_time.sh

```diff
@@ -28,5 +28,8 @@ sed -i 's/"shanghaiTime".*$/"shanghaiTime": '"$CAPELLA_TIME"',/g' $genesis_file
 CANCUN_TIME=$((GENESIS_TIME + (DENEB_FORK_EPOCH * $SLOT_PER_EPOCH * SECONDS_PER_SLOT)))
 echo $CANCUN_TIME
 sed -i 's/"cancunTime".*$/"cancunTime": '"$CANCUN_TIME"',/g' $genesis_file
+PRAGUE_TIME=$((GENESIS_TIME + (ELECTRA_FORK_EPOCH * $SLOT_PER_EPOCH * SECONDS_PER_SLOT)))
```

Comment:
i'm getting this on startup - I wonder if there's an overflow happening?
`INFO [03-13|15:04:47.556] Writing custom genesis block
Fatal: Failed to write genesis block: unsupported fork ordering: cancunTime enabled at timestamp 1710356775, but pragueTime enabled at timestamp 1710356583`

Reply:
Just making sure, did you update `lcli` before running the scripts?

Reply:
can't remember! will try to repro tomorrow

Reply:
On a closer inspection, I'm also getting this issue and it's definitely an overflow. Setting `ELECTRA_FORK_EPOCH` to 9999999 has fixed it. That explains why these scripts never used `18446744073709551615`  :sweat_smile: